src/AspNet.Security.OAuth.JumpCloud/AspNet.Security.OAuth.JumpCloud.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Comment:
The baseline needs to be set to the release this provider will first ship as. Otherwise the baseline will default to `7.0.0`, and we wouldn't have such a package version for JumpCloud.

```suggestion
    <PackageValidationBaselineVersion>7.0.4</PackageValidationBaselineVersion>
    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
```

Reply:
Added the base version